acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/ContentSteps.java

```diff
@@ -223,4 +230,41 @@ public void theUploadIsRejectedWithAnErrorMessage() throws Throwable {
                 contentPage.getErrorMessage(), startsWith(expectedErrorMessage));
         });
     }
+
+    @Given("^I have a number o publications belonging to the same time series$")
```

Comment:
Typo "o" -> "of" in "number o publications", and in method name too.

Reply:
Fixed.